cluster/examples/kubernetes/ceph/object-multisite.yaml

```diff
@@ -27,12 +27,12 @@ spec:
     failureDomain: host
     replicated:
       size: 1
-      requireSafeReplicaSize: true
```

Comment:
We should have the default examples use replica 1, and move these examples with replica 1 to files that have a file suffix of `-test.yaml`